Add 4 BF commands

a2aaron opened the pull request:
+, -, <, and > were added as BF commands. Note that < will not wrap around to the other end of the array if used on the first cell.

a2aaron committed:
Add 4 BF commands

+, -, <, and > were added as BF commands. Note that < will not wrap around to the other end of the array if used on the first cell.

porglezomp commented on brain_friendly.py:
```diff
@@ -0,0 +1,26 @@
+# Current pointer location.
+bf_index = 0
+# Array of byte cells (currently 50).
+bf_array = [0]*50
```
The original says to use 30,000

a2aaron committed:
Change number of cells to 30,000 and rename parse_program()

While there is no offical specication for BF, many interpreters, including the original BF author, use atleast 30,000 cells in memory.


In addition, rename parse_program to eval_program since that's what that function actuall does.

porglezomp commented:
:tada: :tangerine:
